wiki/Beatmap/Pattern/osu!/Jump/en.md

```diff
@@ -2,10 +2,17 @@
 stub: true
 tags:
   - jumps
+  - cross screen jumps
 ---
 
 # Jump
 
 A **jump** is an [osu!](/wiki/Game_mode/osu!) [pattern](/wiki/Beatmap/Pattern) where two [hit objects](/wiki/Gameplay/Hit_object) are spaced far apart on the playfield compared to the spacing between other objects. Jumps usually signal high intensity in a song and they are most commonly snapped to 1/2 ticks on the [timeline](/wiki/Client/Beatmap_editor/Timelines#hit-objects).
 
 Jumps are usually played with snappy movement, as opposed to the smooth movement of [streams](/wiki/Beatmap/Pattern/osu!/Stream).
+
+# Cross Screen Jumps
+
+**Cross Screen Jumps** are a type of [jump](/wiki/Beatmap/Pattern/osu!/Jump) pattern that occur when the distance between two [hit objects](/wiki/Gameplay/Hit_object) is big enough to go across most of the [playfield](/wiki/Client/Playfield).
```

Comment:
The reason why I wanted for CSJ to be specific, or atleast to some extent, is because I feel that saying that it's a jump that goes across the screen is just not a good definition at all, and if so I'd like to give it even if it wouldn't be used in such way.
Since everyone plays on different hardware and settings, there can easily be a situation where a CSJ would not be actually go accros the screen.

**I could easily go from not specifying it in pixels, but I want for it to have more of a strict definition so people who play on different settings and new players can have a easy way to differ them.**

Also there are maps that mostly consist of higher spaced jumps that can be sometimes CSJ, so good luck differing one from another if you can't play the map and don't have any experience and any sort of definition to help you.

Unless it will be a really big requirement to drop it from the wiki, or just not giving it a any sort of a bit more expanded definition, I'm not planning to drop it entirely.

Reply:
there is no one way to differentiate when to use certain adjectives to describe things and that's okay. I think you're getting kind of lost in this quest to define vague terms to the point of not actually giving the reader any valuable info about their real-world usage